src/transmuxer-worker.js

```diff
@@ -8,35 +8,27 @@
  * transmuxer running inside of a WebWorker by exposing a simple
  * message-based interface to a Transmuxer object.
  */
-var
-  muxjs = {},
-  transmuxer,
-  initOptions = {};
-
-importScripts('../node_modules/mux.js/lib/utils/stream.js');
-importScripts('../node_modules/mux.js/lib/utils/exp-golomb.js');
-importScripts('../node_modules/mux.js/lib/mp4/mp4-generator.js');
-importScripts('../node_modules/mux.js/lib/codecs/aac.js');
-importScripts('../node_modules/mux.js/lib/codecs/h264.js');
-importScripts('../node_modules/mux.js/lib/m2ts/m2ts.js');
-importScripts('../node_modules/mux.js/lib/m2ts/caption-stream.js');
-importScripts('../node_modules/mux.js/lib/m2ts/metadata-stream.js');
-importScripts('../node_modules/mux.js/lib/mp4/transmuxer.js');
+let muxjs = require('mux.js');
+let gTransmuxer;
```

Comment:
global, maybe I should just name it globalTransform. wireTransmuxerEvents takes in transmuxer as an argument (what this variable was previously called) which was confusing, and vjsstandard did not like it.